README.markdown

```diff
@@ -2,12 +2,12 @@
 
 ## INSTALLING OR UPGRADING
 ** Please note **: This module is currently undergoing some structural
-maintenance. Please take a look at [https://github.com/jfryman/puppet-nginx/blob/master/docs/hiera.md](https://github.com/jfryman/puppet-nginx/blob/master/docs/hiera.md)
+maintenance. Please take a look at [https://github.com/voxpupuli/puppet-nginx/blob/master/docs/hiera.md](https://github.com/voxpupuli/puppet-nginx/blob/master/docs/hiera.md)
 before upgrading or installing Version 0.1.0 or greater.
 
 [![Puppet
 Forge](http://img.shields.io/puppetforge/v/jfryman/nginx.svg)](https://forge.puppetlabs.com/jfryman/nginx)
```

Comment:
I'm aware, but there's no module on Forge under voxpupuli yet. https://forge.puppet.com/puppet/nginx gets you a 404. Happy to update to change this link also? I'm assuming the new module is uploaded soon enough.